--- .local/lib/python3.10/site-packages/mesonbuild/compilers/cpp.py.org 2023-07-25 14:10:08.388525169 +0900
+++ .local/lib/python3.10/site-packages/mesonbuild/compilers/cpp.py     2023-07-25 14:10:25.468525230 +0900
@@ -212,6 +212,7 @@
             'none', 'c++98', 'c++03', 'c++11', 'c++14', 'c++17', 'c++1z',
             'c++2a', 'c++20', 'gnu++11', 'gnu++14', 'gnu++17', 'gnu++1z',
             'gnu++2a', 'gnu++20',
+            'c++2b',
         ]
         if self.info.is_windows() or self.info.is_cygwin():
             opts.update({

